src/promptflow/tests/sdk_cli_azure_test/recording_utilities/utils.py

```diff
@@ -226,3 +234,10 @@ def is_json_payload_response(response: Dict) -> bool:
     headers = response.get("headers")
     # PFAzureIntegrationTestRecording will lower keys in response headers
     return _is_json_payload(headers, key="content-type")
+
+
+def is_httpx_response(response: Dict) -> bool:
+    # different from other stubs in vcrpy, httpx response uses "content" instead of "body"
```

Comment:
seems aiohttp also uses `content`, but it may not matter here.

Reply:
not sure, but seems `vcrpy` works fine with `aiohttp`, so I guess it can successfully handle `aiohttp` case, so I left the comment for "other stubs in vcrpy"...